package.json

```diff
@@ -3,7 +3,8 @@
   "version": "0.0.0",
   "license": "MIT",
   "scripts": {
-    "postinstall": "nx run-many --target=prisma-generate --all"
+    "postinstall": "nx run-many --target=prisma-generate --all",
+    "prepare": "husky install"
```

Comment:
We'll need to document, that `npm run prepare` should be ran by new users.

Reply:
`prepare` is run automatically after install, so this should happen automatically.
However, we should update this to `husky install || true` so that install doesn't fail in environments that don't have git like in deployments

Reply:
Good to know!